packages/trader/src/App/Components/Layout/Footer/toggle-settings.jsx

```diff
@@ -1,11 +1,13 @@
-import classNames   from 'classnames';
-import PropTypes    from 'prop-types';
-import React        from 'react';
-import { Modal }    from 'App/Components/Elements/modal.jsx';
-import UILoader     from 'App/Components/Elements/ui-loader.jsx';
-import Lazy         from 'App/Containers/Lazy';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Modal } from 'deriv-components';
+import UILoader from 'App/Components/Elements/ui-loader.jsx';
+import Lazy from 'App/Containers/Lazy';
```

Comment:
Please align on `from`

Reply:
@Mahboobeh-binary This conversation was resolved, but we didn't talk about it nor was the change made. Do I need to clear cache?